dipy/denoise/denspeed.pyx

```diff
@@ -108,9 +103,10 @@ def _nlmeans_3d(double[:, :, ::1] arr, double[:, :, ::1] mask,
 
     # move the block
     with nogil, parallel():
-        for i in prange(B + P, I - B - P, schedule="dynamic"):
```

Comment:
And results are identical?
On Jun 24, 2016 9:03 AM, "Riddhish Bhalodia" <notifications@github.com>
wrote:

> In dipy/denoise/denspeed.pyx
> <https://github.com/nipy/dipy/pull/1088#discussion_r68420578>:
>
> > @@ -108,9 +103,10 @@ def _nlmeans_3d(double[:, :, ::1] arr, double[:, :, ::1] mask,
> >
> >      # move the block
> >      with nogil, parallel():
> > -        for i in prange(B + P, I - B - P, schedule="dynamic"):
>
> Yes looking at it. compared speeds, the schedule = "dynamic" makes a
> difference!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nipy/dipy/pull/1088/files/c6068c3877c0dd6a2a7c15dac27ebe21c50a3d9c#r68420578>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHPNqdFtD0XTBIGwODNvA84dFBwfpd2ks5qO_--gaJpZM4I95WC>
> .
>